components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -55,10 +55,31 @@ if err := k.CreateSubscription("my-sub", namespace, channelName, &uri); err != n
 return
 */
 
+// Use it cause it should be mocked
+type KnativeLibIntf interface {
+	GetChannel(name string, namespace string) (*evapisv1alpha1.Channel, error)
```

Comment:
Can we pass all functions(s) arguments by reference to avoid copying?

Reply:
The new interface exposes the same method as the type KnativeLib. It is needed to made possible to mock this interface for unit tests.
The only "string" parameters are "name" and "namespace" (like in the type KnativeLib)